Automattic/course
# Add styles for link in different states

Implements https://github.com/Automattic/sensei-theme/issues/4

### Changes proposed in this pull request

Added styles for different link states. Added the styles via CSS now, because if we add the styles for state pseudo-class selectors using `theme.json`, we also need to add the Gutenberg plugin.

### Testing instructions

- Go to the Create or Edit screen of a Page
- Add a paragraph block
- Add a URL in the paragraph block
- Save the page and check the page in the frontend
- Make sure the link shows the same behavior for active, focus and click as the attached video and issue

### Screenshots

https://user-images.githubusercontent.com/6820724/195327099-c0f09852-222c-4499-a969-63451e689ae1.mov
